app/views/unattended/provisioning_templates/PXELinux/autoyast_default_pxelinux.erb

```diff
@@ -16,6 +16,12 @@ if host_param('http-proxy') && host_param('http-proxy-port')
 elsif host_param('http-proxy')
   extra_args << "proxy=http://" + host_param('http-proxy')
 end
+subnet = @host.subnet
+unless subnet.respond_to?(:dhcp_boot_mode?) and subnet.dhcp_boot_mode?
+  extra_args << "useDHCP=0"
+  extra_args << "netsetup=-dhcp"
+  extra_args << "ifcfg=*=#{@host.primary_interface.ip}/#{@host.primary_interface.subnet.cidr},#{@host.primary_interface.subnet.gateway},#{@host.primary_interface.subnet.dns_primary},#{@host.domain}"
```

Comment:
Wouldn't it be better to use the provision interface like in https://github.com/theforeman/foreman/blob/12bcc1190e16c571fe5d7107dc6050149752d127/app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb#L18 ?

Reply:
I don't know, both the AutoYaST Provision template as well as AutoYaST iPXE use primary_interface as well

https://github.com/theforeman/foreman/blob/12bcc1190e16c571fe5d7107dc6050149752d127/app/views/unattended/provisioning_templates/provision/autoyast_sles_default.erb#L75-L91

https://github.com/theforeman/foreman/blob/cc395951bf21e738a33e35df9e28e1a25ca4bc51/app/views/unattended/provisioning_templates/iPXE/autoyast_default_ipxe.erb#L35